File: Lab1/JAVA/q2.java
import java.util.Scanner;

public class q2 {

    static int sum(int n) {
        int s = 0, d;
        while (n > 0) {
            d = n % 10;
            s += d;
            n = n / 10;
        }
        return s;
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n;
        System.out.print("Enter a number: ");
        n = in.nextInt();
        System.out.println("Sum: " + sum(n));
    }
}

File: Lab1/CPP/q2.cpp
#include<iostream>
#include<conio.h>
using namespace std;

int sum(int n)
{
    int s=0,d;
    while(n>0)
    {
        d=n%10;
        s+=d;
        n=n/10;
    }
    return s;
}

int main()
{
    int n;
    cout<<endl<<"Enter a number: ";
    cin>>n;
    cout<<"Sum: "<<sum(n)<<endl<<endl;
    return 0;
}
